packages/esm-patient-chart-app/translations/km.json

```diff
@@ -102,32 +102,32 @@
   "Programs dashboard": "ផ្ទាំងគ្រប់គ្រងកម្មវិធី",
   "provider": "អ្នកផ្តល់សេវា",
   "quantity": "បរិមាណ",
-  "queueAddedSuccessfully": "Patient has been added to the queue successfully.",
-  "queueEntryError": "Error adding patient to the queue",
+  "queueAddedSuccessfully": "បានបន្ថែមអ្នកជំងឺជាជួរដោយជោគជ័យ",
+  "queueEntryError": "មានកំហុសក្នុងការបន្ថែមអ្នកជំងឺទៅជួរ",
   "recommended": "អនុសាសន៍",
   "record": "Record",
```

Comment:
Translation pending for Record

Reply:
As per @kdaud, the translation was not provided for this key, hence as soon as this is provided, a follow up PR will be created.
Thanks @kdaud !